userguide/virtualmachines.rst

```diff
@@ -663,107 +663,88 @@ Start the Docker Host
+   --- 8.8.8.8 ping statistics ---
+   5 packets transmitted, 5 packets received, 0% packet loss
+   round-trip min/avg/max = 17.343/17.441/17.502 ms
 
-.. note:: If the error :literal:`Cannot connect to the Docker daemon`
-   is shown, run :command:`sudo dockerd`. Then give the
-   :command:`sudo docker run` command above again.
 
+If :command:`ping` returns an error, adjust the VM
+:ref:`Network Interface <vms-network-interface>` and reboot the VM.
 
-Installation time varies with processor and network connection speed,
-but typically takes a few minutes. After the process finishes and a
-command prompt is shown, type this command:
-
-.. code-block:: none
-
-   ifconfig eth0 | grep 'inet addr'
+Download and install the Rancher system with
+:command:`sudo docker run -d --restart=unless-stopped -p 8080:8080 rancher/server`.
 
+If :literal:`Cannot connect to the Docker daemon` shows, run
+:command:`sudo dockerd` and try the command again. Installation time
+varies with processor and network connection speed.
 
-The first value is the IP address of the Rancher server. Enter the IP
-address and port :literal:`8080` as the URL in a web browser. For
-example, if the IP address was :literal:`10.231.3.208`, enter
-:literal:`10.231.3.208:8080` as the URL in the web browser.
+When the command prompt is available, enter
+:command:`ifconfig eth0 | grep 'inet addr'`. Copy the
+:literal:`inet addr:` IP address. Enter the IP address appended with
+:literal:`:8080` into a web browser to connect to the Rancher server
+|web-ui|. For example, if the IP address was :literal:`10.231.3.208`,
+:literal:`10.231.3.208:8080` is entered in a browser.
 
 The Rancher server takes a few minutes to start. The web browser might
-show a connection error while the Rancher |web-ui| is still starting. If
-the browser shows a :literal:`connection has timed out` or a similar
-error, wait one minute and try again.
+show a connection error while the Rancher |web-ui| starts. If the
+browser shows a :literal:`connection has timed out` error, wait one
+minute and try again.
 
-In the Rancher |web-ui|, click :guilabel:`Add a host`, ensure the radial
-:guilabel:`This site's address` button is set, and click
-:guilabel:`Save`. Follow the instructions that now display and run the
+When the Rancher |web-ui| loads, click :guilabel:`Add a host`.
+Ensure the radial :guilabel:`This site's address` button is set and
+click :guilabel:`Save`.
+
+Follow the instructions that now display and run the
 :command:`sudo docker run --rm --privileged -v` command in the Docker
-Host Serial shell. After the command runs a message displays
-:literal:`Launched Rancher Agent:`. Refresh or go to the
-:guilabel:`Hosts` page of the Rancher |web-ui| to confirm the Docker
-Host displays in the |web-ui|. Rancher is now configured and ready for
-use.
+Host Serial shell. :literal:`Launched Rancher Agent:` displays when the
+command is successful.
+
+Go to the :guilabel:`Hosts` page of the Rancher |web-ui| to confirm the
```

Comment:
Point out that this is a confirmation check. (Short sentences.)
0. Verify that the configuration is complete by:
1. Go to Rancher web interface, click Hosts.
2. Verify presence of (something) (literal? IP address?  hostname?)
3. If present, configuration is complete.